entity-framework/core/providers/sqlite/spatial.md

```diff
@@ -13,7 +13,7 @@ This page includes additional information about using spatial data with the SQLi
 
 On Windows, the native mod_spatialite library is distributed as a NuGet package dependency. Other platforms need to install it separately. This is typically done using a software package manager. For example, you can use APT on Debian and Ubuntu; and Homebrew on MacOS.
 
-``` sh
+``` bash
```

Comment:
```suggestion
```bash
```

Reply:
Sure, I'll normalize. I was surprised to see that the majority had a space